examples/tests/malloc_test01/Makefile

```diff
@@ -6,6 +6,8 @@ TOCK_USERLAND_BASE_DIR = ../../..
 # Which files to compile.
 C_SRCS := $(wildcard *.c)
 
+STACK_SIZE       = 4096
```

Comment:
I am shocked this requires a 4kB stack on RISC-V -- all it does is malloc a 5, 15, and 20 byte buffer and iterate through each before printing a short string. 4kB is the stack size for the entire kernel on some boards.

I wonder if something else is going on here and changing the stack size is somehow masking the actual problem

Reply:
Well...it also prints "success" if it gets that far. And I'm pretty sure that is what the issue is.